packages/react-ui-components/src/Frame/frame.js

```diff
@@ -13,6 +14,22 @@ export default class Frame extends PureComponent {
 
     handleReference = ref => {
         this.ref = ref;
+    };
+
+    componentDidMount() {
+        this.componentDidUpdate();
```

Comment:
Please not like this! Extract the logic that should be executed both on mount and update into another method and call it from two places.

Reply:
ok if you like ^^